lib/gpu/lal_dpd_coul_slater_long.cu

```diff
@@ -3,7 +3,7 @@
 //                             -------------------
 //                           Trung Dac Nguyen (ORNL)
```

Comment:
@Eddy-Barraud please add your name ahead of mine, and replace ORNL with U Chicago.